.github/workflows/pytest.yml

```diff
@@ -31,7 +31,26 @@ jobs:
     - name: Setup xvfb
       run: |
         sudo apt-get update
-        sudo apt-get install -y xvfb libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xinput0 libxcb-xfixes0
+        sudo apt-get install -y xvfb \
+            libxkbcommon-x11-0 \
+            libxcb-icccm4 \
+            libxcb-image0 \
+            libxcb-keysyms1 \
+            libxcb-randr0 \
+            libxcb-render-util0 \
+            libxcb-xinerama0 \
+            libxcb-xinput0 \
+            libxcb-xfixes0 \
+            pkg-config \
```

Comment:
Consider "Setup xvfb" as a unit that does not need to be changed.
Dependencies are installed in "Install Python dependencies".
To not pollute the yml the packages.list gets parsed. I think it would be smart to add the additional imports to the list or delete the lists and put everything in the yml file. Atm we are trying to install some packages twice.

Reply:
Personally, I'd rather move them out of the workflow itself into packages.list and add that to the Makefile as a way of installing things.

Reply:
Hey @meyerkev I also think its better to have them in one place and thank you for sorting btw

Reply:
@p5k369 would you like to ``Resolve`` this conversation.

Reply:
@eoyilmaz I dont think that an outdated conversations can be resolved. I think this happens if the lines someone commented on are gone.